.github/workflows/make_wheel.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         cfg:
           - { name: 'Linux LLVM+libstdc++', os: 'ubuntu-20.04', platform: 'manylinux_2_31_x86_64', cc: clang, cxx: clang++, config: }
           - { name: 'MacOS LLVM+libc++', os: 'macos-10.15', platform: 'macosx_10_15_x86_64', cc: clang, cxx: clang++, config: --config=libc++ }
+          - { name: 'MacOS LLVM+libc++', os: 'macos-11', platform: 'macosx_10_16_x86_64', cc: clang, cxx: clang++, config: --config=libc++ }
```

Comment:
Is this platform tag correct?

Reply:
That's what gets built by the wheel target, and (in my own fork) I've confirmed this is used by `pip install`.

FYI: https://eclecticlight.co/2020/07/21/big-sur-is-both-10-16-and-11-0-its-official

Reply:
Ah right, that's the resulting ABI tag. Yeah, that makes sense.